packages/utils/popup-manager.ts

```diff
@@ -88,7 +96,7 @@ const PopupManager: IPopupManager = {
   },
 
   nextZIndex() {
-    return ++PopupManager.zIndex
+    return this.getInitialZIndex() + ++this.zIndex
```

Comment:
```suggestion
return `${this.getInitialZIndex()}${this.zIndex}`
```

Reply:
zindex is a number.

Reply:
So this is indeed the addition of two numbers, and template strings cannot be used.